app/scripts/metamask-controller.js

```diff
@@ -1794,6 +1795,11 @@ export default class MetamaskController extends EventEmitter {
         allowedEvents: ['NameController:stateChange'],
       }),
     }).init();
+
+    setupAccountLabelsPetnamesBridge(
```

Comment:
Apologies if this was not yet refactored, but we need to update this to create an instance of the new `AccountIdentitiesPetnamesBrdige` and invoke `init`.